### PR TITLE
fix: [#183536833] inline error for missing signature for dakota nexus

### PR DESCRIPTION
**Pull request by somabadri**
## Description

Fix to inline error for missing signature for dakota nexus.

### Ticket

[#183536833](https://www.pivotaltracker.com/story/show/183536833)

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation, if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: web/src/components/tasks/business-formation/contacts/Signatures.test.tsx [hunk 1]
```diff
@@ -47,59 +47,65 @@ describe("Formation - Signatures", () => {
...
-          {
-            name: "V",
-            signature: true,
-          },
-          legalStructureId
-        ),
-      ]);
-    });
+    it.each([...legalStructureIds])(
+      "adds additional signer for legalStructure is %s",
```
Comment (AnJuHyppolite) [inline web/src/components/tasks/business-formation/contacts/Signatures.test.tsx:51]: [pebble] Can we update the description to `"adds additional signer if legalStructure is %s"`

Comment (somabadri) [reply web/src/components/tasks/business-formation/contacts/Signatures.test.tsx]: done!

File: web/src/components/tasks/business-formation/contacts/Signatures.test.tsx [hunk 2]
```diff
@@ -250,24 +259,62 @@ describe("Formation - Signatures", () => {
...
+          return screen.queryByText(Config.formation.fields.signers.errorInlineNameAndSignature);
         };
         const signerTitleErrorType = (): HTMLElement | null => {
           return screen.queryByText(Config.formation.fields.signers.errorInlineSignerTitle);
         };
-        expect(singerNameErrorType()).toBeInTheDocument();
+        expect(singerNameAndSignatureErrorType()).toBeInTheDocument();
         expect(signerTitleErrorType()).toBeInTheDocument();
       }
     );
```
Comment (AnJuHyppolite) [inline web/src/components/tasks/business-formation/contacts/Signatures.test.tsx:276]: This appears to be the same test above, lines 171-200.

File: web/src/components/tasks/business-formation/contacts/Signatures.test.tsx [hunk 3]
```diff
@@ -163,7 +169,7 @@ describe("Formation - Signatures", () => {
...
 
     it.each([...legalStructureIds])(
-      "displays inline error when signer name longer than 50 chars for %s",
+      "displays inline error when signer name longer than 50 chars when legalStructure is %s",
```
Comment (AnJuHyppolite) [inline web/src/components/tasks/business-formation/contacts/Signatures.test.tsx:172]: This test seems similar to the one below (see comment).

File: web/src/components/tasks/business-formation/contacts/Signatures.test.tsx (hunk 2 above)
Comment (somabadri) [reply web/src/components/tasks/business-formation/contacts/Signatures.test.tsx:276]: They are testing different things, the one above is testing that the 'greater than 50 characters' error shows up in line and this is testing that name and signature  is missing and add title error shows up, they are also for different users, poppy vs dakota nexus

**Pull request merged**